script/upload_to_aptly

```diff
@@ -19,22 +19,22 @@ set -exu
 # 12/29/2018    Nightly       0x0B31DBA06A8A26F9    9228DBCE20DDE5EC46488DE90B31DBA06A8A26F9
 
 ### The first argument is the S3 bucket, the second is the GPG key fingerprint
-OPTION=${1:-brave-browser-apt-staging-nightly}
-GPG_KEY_ID=${2:-D8BAD4DE7EE17AF52A834B2D0BB75829C2D4E821} # Defaults to New Brave Software signing key
+OPTION=${1:-brave-browser-apt-staging-release-test}
+GPG_KEY_ID=${2:-9BD198A2D848C482E3550697A8580BDC82D3DC6C} # Defaults to New Brave Software signing key
```

Comment:
Maybe we shouldn't set a default key, that way we don't have to update this variable when we rotate the subkeys.

Reply:
This is a good solution, I copied the scripts from the laptop-updates repo so basically kept the style the same. But this is a good time to make more substantive changes.

Reply:
Resolved by not setting a default key, then erroring if key is not passed